python/feathub/table/table_descriptor.py

```diff
@@ -54,6 +54,7 @@ def __init__(
         keys: Optional[List[str]],
         timestamp_field: Optional[str],
         timestamp_format: str = "epoch",
+        attributes: Optional[Dict[str, str]] = None,
```

Comment:
Would it be better to name it `extra_props` for consistency with `FlinkProcessor.props` and `FeatureTable.data_format_properties`?